src/app/feed/feed-lightbox/feed-lightbox.component.spec.ts

```diff
@@ -1,9 +1,79 @@
 /* tslint:disable:no-unused-variable */
 import { FeedLightboxComponent } from './feed-lightbox.component';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+
+@Component({
+	selector: 'feed-linker',
+	template: ''
+})
+class FeedLinkerStubComponent {
+	@Input() text: string;
+	@Input() hashtags: string[] = new Array<string>();
+	@Input() mentions: string[] = new Array<string>();
+	@Input() links: string[] = new Array<string>();
+	@Input() unshorten: Object = {};
+	@Input() useAll = false;
+	@Output() showed: EventEmitter<boolean> = new EventEmitter<boolean>();
+}
 
 describe('Component: FeedLightbox', () => {
+
+	beforeEach(() => {
+		TestBed.configureTestingModule({
+			imports: [
+				RouterTestingModule
+			],
+			declarations: [
+				FeedLinkerStubComponent,
+				FeedLightboxComponent
+			]
+		});
+	});
+
 	it('should create an instance', () => {
-		const component = new FeedLightboxComponent();
+		const fixture = TestBed.createComponent(FeedLightboxComponent);
+		const component = fixture.debugElement.componentInstance;
 		expect(component).toBeTruthy();
 	});
+
+	it('should use feedItem for various function', async(() => {
+		const fixture = TestBed.createComponent(FeedLightboxComponent);
+		const component = fixture.debugElement.componentInstance;
+
+		component.feedItem = {
+			images : [],
+			screen_name : '@fossasia',
+			user : {
+				name : 'testUser'
+			},
+			text : 'testText',
+			retweet_count : 20,
+			favourites_count : 25,
+			created_at : new Date(),
+			videos : []
+		};
+		component.ttt();
+		expect(component.datetime).toBe('now');
```

Comment:
We actually need something meaningful in tests. Test cases should cover the actual codebase to test the important parts of project like API calls, data processing, etc. Only expecting each component and their parts to be ```truthy``` on project run is not something fruitful. Please try to add more meaningful cases/ add something which really tests something.

Reply:
Will definitely try to add something more meaningful to this component

Reply:
@shreyanshdwivedi I am talking for overall in general, not only for this component.